automated/linux/aklite-apps/aklite-apps.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (C) 2021 Foundries.io Ltd.
+
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+APPS=""
+DEFAULT_APPS=""
+LOOP_APPS=""
+APPS_ARG=""
+
+usage() {
+    echo "\
+    Usage: $0 -a <app1,app2> -d <app1,app2>
+
+    -a <app1,app2>
+        Comma separated list of apps to start
+    -d <app1,app2>
+        Comma separated list of default apps to start
+        Conflicts with -a
```

Comment:
it doesn't conflicts it just ignores the defaults right?
Maybe explain that.

Reply:
You're right. I rephrased the help message and also added some text to .yaml file. This should get converted into documentation.